Face/Identify/Program.cs

```diff
@@ -0,0 +1,218 @@
+
+            List<Guid> faceIds = new List<Guid>();
+            // Read image file. 
+            using (FileStream stream = new FileStream(Path.Combine("Images", SourceImageFileName), FileMode.Open)) 
+            { 
+                // Detect faces from image stream. 
+                IList<DetectedFace> detectedFaces = await Client.Face.DetectWithStreamAsync(stream); 
+                if (detectedFaces == null || detectedFaces.Count == 0) 
+                { 
+                    Console.WriteLine($"[Error] No face detected from image `{SourceImageFileName}`."); 
+                    return; 
+                } 
+
+                Console.WriteLine($"{detectedFaces.Count} faces detected from image `{SourceImageFileName}`."); 
+                if (detectedFaces[0].FaceId == null) 
+                { 
+                    Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for Identify purpose.");
+                    return; 
+                }
+
+                // Add detected faceId to faceIds.
+                foreach (var face in detectedFaces)
+                {
+                    faceIds.Add(face.FaceId.Value);
+                }
+            }
+
+            // Identify example for get person from the person group with candidate confidence. 
+            var identifyResults = await Client.Face.IdentifyAsync(faceIds, GroupId);
+            if (identifyResults == null)
+            {
+                Console.WriteLine($"No people found in the group the same as the {SourceImageFileName}.");
+                return;
+            }
+
+            foreach (var identifyResult in identifyResults)
+            {
+                Person person = await Client.PersonGroupPerson.GetAsync(GroupId, identifyResult.Candidates[0].PersonId);                
+                Console.WriteLine($"Person from {SourceImageFileName} & group person '{person.Name}' are of the same (Positive) person, confidence: {identifyResult.Candidates[0].Confidence}.");
+            }
+
+            // Delete the person group.
+            await Client.PersonGroup.DeleteAsync(GroupId);
+            Console.WriteLine($"{Environment.NewLine}Delete the person group {GroupId}{Environment.NewLine}"); 
+        }
+
+        public static async Task Identify_in_LargePersonGroup()
+        {
+            // Create a large person group.
+            string GroupId = Guid.NewGuid().ToString();
+            Console.WriteLine($"Create a large person group ({GroupId}) {Environment.NewLine}");
+            await Client.LargePersonGroup.CreateAsync(GroupId, GroupId);
+
+            foreach (var targetImageFileName in TargetImageFileNames)
+            {
+                // Read image file. 
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open))
+                {
+                    // Create a person group person.
+                    Person p = new Person();
```

Comment:
Person p = new Person { Name = targetImageFileName, UserData = "Person for sample" };

Reply:
done